### DISCUSSION TITLE
Add an option to allow to run as administrators on boot

### DISCUSSION THREAD

**Discussion by NinelieZ** [File-New-Project/EarTrumpet#1461] [Ideas]
While active windows run as admin, the mouse wheel hover on the icon does not work. After runned Ear Trumpet as admin this problem is gone. Could you please add an option to allow us starting Ear Trumpet as admin ?

**Comment by riverar** [comment]
![image](https://github.com/File-New-Project/EarTrumpet/assets/475132/6d5ad81d-4fd1-46c3-8f53-c32984b8faba)

If you right-click EarTrumpet in Start > Run as administrator, it should work.

I filed a bug on the Windows team too https://aka.ms/AAmvqc2.

EDIT: Oh you want it to startup automatically hmm.

**Comment by NinelieZ** [comment]
Yeah, I want it to startup automatically. 
Almost forgot Feedback Hub, as I submitted some small bug and no response years later. Such as Windows Search enhanced mode doesn't work on Win 11 about two year now. I tried about four different devices and all of them doesn't work. _(:зゝ∠)_
